mms/tests/integration_tests/test_utils.py

```diff
@@ -68,64 +75,76 @@ def cleanup(tmpdir):
     shutil.rmtree(str(tmpdir))
 
 
+def create_model_onnx_zoo(download_dir, url, onnx_model):
+    _download_file(download_dir, url[onnx_model])
+    _download_file(
+        download_dir,
+        "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/signature.json")
+    _download_file(
+        download_dir,
+        "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/synset.txt")
+    model_tar = '{}/{}.tar.gz'.format(download_dir, onnx_model)
+    tar = tarfile.open(model_tar, "r:*")
+    tar.extractall(path=download_dir)
+    tar.close()
+    model_dir = '{}/{}'.format(download_dir, onnx_model)
+    model_path = os.path.join(model_dir, 'model.onnx')
+    new_path = os.path.join(
+        model_dir, '{}.onnx'.format(onnx_model))
+    os.rename(model_path, new_path)
+    shutil.move(new_path, download_dir)
+
+
+def create_model_onnx_mxnet_zoo(download_dir, url, onnx_model):
+    _download_file(download_dir, url[onnx_model])
+    _download_file(
+        download_dir,
+        "https://s3.amazonaws.com/model-server/models/" +
+        onnx_model +
+        "/signature.json")
+    _download_file(
+        download_dir,
+        "https://s3.amazonaws.com/model-server/models/" +
+        onnx_model +
+        "/synset.txt")
+
+
+def export_onnx_models(tmpdir, onnx_source_model_zoo, url, onnx_model):
+    download_dir = tmpdir + '/mxnet_model_server'
+    os.mkdir(download_dir)
+    if onnx_source_model_zoo:
+        create_model_onnx_zoo(download_dir, url, onnx_model)
+    else:
+        create_model_onnx_mxnet_zoo(download_dir, url, onnx_model)
+
+    print('model files prepared for model {} '.format(onnx_model))
+    # Export the model.
+    print("Exporting the mxnet model server model...")
+    subprocess.check_call(['mxnet-model-export',
+                           '--model-name',
+                           onnx_model,
+                           '--model-path',
+                           download_dir],
+                          cwd=download_dir)
+    shutil.move('{}/{}.model'.format(download_dir, onnx_model), tmpdir)
+    for root, dirs, files in os.walk(download_dir):
+        for f in files:
+            os.unlink(os.path.join(root, f))
+        for d in dirs:
+            shutil.rmtree(os.path.join(root, d))
+    shutil.rmtree(download_dir)
+
+
 def create_model(
         tmpdir,
         url,
         onnx_model,
```

Comment:
why is this named onnx_model when it is used in both branches of is_onnx_model test?

bigger question: why do we (still) need 3 parameters related to the onnx-ness of a model?